packages/sdks/mitosis.config.cjs

```diff
@@ -942,6 +948,30 @@ const QWIK_ONUPDATE_TO_USEVISIBLETASK = () => ({
   },
 });
 
+/**
+ * Updates the placement of the style element in the BlocksWrapper component. Injects it at the top of the fragment.
+ * This is necessary because when we run the inlined script, we only have access to previous sibling elements for deletion.
+ */
+const UPDATE_PLACEMENT_OF_STYLE_ELEMENT_IN_BLOCKS_WRAPPER = () => ({
```

Comment:
if we can replace this with mitosis logic here: https://github.com/BuilderIO/mitosis/blob/0a49334818422d76efd89f9cdde1af5ab3bedcef/packages/core/src/generators/react/generator.ts#L423-L434

`state.stylesLocation: 'top' | 'bottom'`

Reply:
done!